CMakeLists.txt

```diff
@@ -160,6 +160,9 @@ if(NOT WIN32 AND ${hip_VERSION_FLAT} GREATER 600241132)
   add_compile_options("SHELL: -mllvm -amdgpu-function-calls=false")
 endif()
 
+find_package( composable_kernel 1.0.0 REQUIRED PATHS $ENV{CK_DIR}/lib/cmake /opt/rocm /opt/rocm/ck COMPONENTS device_contraction_operations device_reduction_operations device_other_operations)
```

Comment:
hipTensor historically relied on CK package for static headers, and only included in components that used CK as a backend in case we needed to switch it out ever. Not sure if we want to drag it into the whole project.

We want to ensure hipTensor libraries deployment doesn't depend on CK shared libraries. Is this still the case?